File: desktop/src/com/flappy/kotlin/game/desktop/DesktopLauncher.java
package com.flappy.kotlin.game.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.flappy.kotlin.game.FlappyKotlinGame;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		config.width = FlappyKotlinGame.WIDTH;
		config.height = FlappyKotlinGame.HEIGHT;
		config.title = FlappyKotlinGame.TITLE;

		new LwjglApplication(new FlappyKotlinGame(), config);
	}
}

File: desktop/src/com/flappy/kotlin/game/desktop/DesktopLauncher.kt
package com.flappy.kotlin.game.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.flappy.kotlin.game.FlappyKotlinGame

object DesktopLauncher {
    @JvmStatic
    fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration()
        config.width = FlappyKotlinGame.WIDTH
        config.height = FlappyKotlinGame.HEIGHT
        config.title = FlappyKotlinGame.TITLE

        LwjglApplication(FlappyKotlinGame(), config)
    }
}
